File: C/2 Seleção I/mumia.c
#include <stdio.h>
#include <string.h>

int main(){
    char nome[100], ans[100];
    int idade;
    scanf("%s %d", nome, &idade);

    if(idade<12)strcpy(ans, "crianca");
    else if(idade<18) strcpy(ans,"jovem");
    else if(idade<65) strcpy(ans,"adulto");
    else if(idade<1000) strcpy(ans,"idoso");
    else strcpy(ans, "mumia");

    printf("%s eh %s\n", nome, ans);
}

File: Fup is Fun/Python/2 Operações/mumia.py
nome = input()
idade = int(input())

ans = ""
if idade<12: ans = "crianca"
elif idade<18: ans = "jovem"
elif idade<65: ans = "adulto"
elif idade<1000: ans = "idoso"
else: ans = "mumia"

print(f"{nome} eh {ans}")